buf.yaml

```diff
@@ -31,9 +31,3 @@ lint:
 breaking:
   use:
     - FILE
-  ignore:
-    - tendermint
-    - gogoproto
-    - cosmos_proto
-    - google
-    - confio
```

Comment:
For other reviewers ref: https://buf.build/docs/configuration/v1/buf-yaml#ignore-1
Looks like this block ignored breaking changes from these directories but all our protos are defined in proto/celestia or proto/cosmos so I don't expect it to have done anything